modules/intentIqIdSystem.js

```diff
@@ -315,23 +330,23 @@ export const intentIqIdSubmodule = {
     PARTNER_DATA_KEY = `${FIRST_PARTY_KEY}_${configParams.partner}`;
 
     const allowedStorage = defineStorageType(config.enabledStorageTypes);
+    partnerData = tryParse(readData(PARTNER_DATA_KEY, allowedStorage)) || {};
 
     let rrttStrtTime = 0;
-    let partnerData = {};
     let shouldCallServer = false;
     FIRST_PARTY_KEY_FINAL = `${FIRST_PARTY_KEY}${siloEnabled ? '_p_' + configParams.partner : ''}`;
     const cmpData = getCmpData();
     const gdprDetected = cmpData.gdprString;
     firstPartyData = tryParse(readData(FIRST_PARTY_KEY_FINAL, allowedStorage));
-    const isGroupB = firstPartyData?.group === WITHOUT_IIQ;
+    let actualABGroup = defineABTestingGroup(partnerData?.terminationCause, configParams.abPercentage);
```

Comment:
This will need a mechanism to pass the defined abgroup from the ID Module to the Analytics module. For new users, and without a tc from your server, a group will be assigned here at random. Later in the Analytics BidWon event, it will reroll again at random. A user can flip between groups for the same auction.

Does that sound correct to you or am I misunderstanding how the two are being called and using `defineABTestingGroup`?